specs/spec.js

```diff
@@ -24,10 +20,139 @@ describe('Database seeded', () => {
 
   it('Database seeded with 10 Sellers', () => {
     let sellerCount = 0;
-    Seller.count()
+    Seller.countDocuments()
       .then((count) => {
         sellerCount = count;
         expect(sellerCount).to.equal(10);
       });
   });
 });
+
+describe('Test API routes', () => {
+  const priceOptions = [9.99, 19.99, 29.99, 39.99, 49.99, 59.99, 99.99];
+  const sellerOptions = [
+    'Tortor',
+    'Nisl',
+    'Consequat',
+    'Vulputate Ut',
+    'Adipiscing Elit Pellentesque',
+    'Fames Nunc',
+    'Ultricies',
+    'Morbi',
+    'Odio',
+    'Sit Amet',
+  ];
+
+  it('Fetch product pricing data with productId, test /product/prices route', () => {
+    request.get('/api/product/prices?productId=1')
+      .expect((res) => {
+        expect(res.body[0].productId).to.equal(1);
+        expect(res.body.length).to.equal(1);
+      })
+      .end((err) => {
+        if (err) throw err;
+      });
+  });
+
+  it('Fetch product pricing data, test /product/prices route', () => {
+    request.get('/api/product/prices')
+      .expect((res) => {
+        expect(priceOptions.includes(res.body[0].seller[0].price)).to.equal(true);
+        expect(res.body.length).to.equal(100);
+      })
+      .end((err) => {
+        if (err) throw err;
+      });
+  });
+
+  it('Fetch product seller data, test /product/sellers route', () => {
+    request.get('/api/product/sellers')
+      .expect((res) => {
+        expect(sellerOptions.includes(res.body[0].name)).to.equal(true);
+      })
+      .end((err) => {
+        if (err) throw err;
+      });
+  });
+
+  it('Fetch product quotes data, test /product/quotes route', () => {
+    request.get('/api/product/quotes')
+      .expect((res) => {
+        expect(priceOptions.includes(res.body[0].seller[0].price)).to.equal(true);
+        expect(sellerOptions.includes(res.body[0].seller[0].name)).to.equal(true);
+      })
+      .end((err) => {
+        if (err) throw err;
+      });
+  });
+});
```

Comment:
Are you able to test the failures?

Reply:
.end((err) => {
        if (err) throw err;
      });
In my view above handles basic error scenarios. This takes core of basic tests for now.

Reply:
It does not make sense to me handling an error scenario for a test that is meant to pass. I'm going to imply that 'for now' means the tests will be created later.